requirements.txt

```diff
@@ -185,7 +185,7 @@ tornado==6.3.3
 tqdm==4.66.1
 traitlets==5.10.0
 transformers==4.33.2
-triton==2.0.0
+# triton==2.0.0
```

Comment:
Is this required?

Reply:
i presume this is also related to nvidia gpu. so same explanation as [above](https://github.com/BaranziniLab/KG_RAG/pull/34/files#r1669925261)